paramiko/hostkeys.py

```diff
@@ -25,11 +25,11 @@
 
 from paramiko.py3compat import b, u, encodebytes, decodebytes
 
-try:
-    from collections import MutableMapping
-except ImportError:
+try:                                                    # pragma: no cover
+    from collections import MutableMapping              # pragma: no cover
+except ImportError:                                     # pragma: no cover
     # noinspection PyUnresolvedReferences
-    from UserDict import DictMixin as MutableMapping
+    from UserDict import DictMixin as MutableMapping    # pragma: no cover
```

Comment:
I don't think this "code coverage ignore" is worth it - just take the code coverage report with a grain of salt, understand that it's just a tool, and 100% code coverage doesn't actually guarantee everything is tested anyway.

(Side note: all the coveralls PR comments are annoying. I've seen other projects that have coveralls instead put the score as an extra commit status right next to the CI result, which I prefer.)